README.md

```diff
@@ -106,73 +259,49 @@ See [Model Zoo Overview](./docs/model_zoo_overview.md) for more infomation.
+    * [Detailed Configuration File](./docs/design/use/use.md)
+    * [Create Your Own Model](./docs/design/create/add_new_model.md)
+*  Pull Request
+    * [PR Tutorial](./docs/pr/pr/pr.md)
+    * [PR Style](./docs/pr/pr/style_cn.md)
+
 * [Static Graph Version](./docs/static/static.md)
+* [Community](#Community)
 * [FAQ](./docs/faq/faq/faq.md)
 
-## Installation
-
-#### step 1. Install PaddlePaddle
+## Practical Projects
+  * [Interactive Segmentation](./EISeg)
+  * [Image Matting](./contrib/Matting)
+  * [PP-HumanSeg](./contrib/PP-HumanSeg)
+  * [Cityscapes SOTA](./contrib/CityscapesSOTA)
+  * [Panoptic Segmentation](./contrib/PanopticDeepLab)
+  * [CVPR Champion Solution](./contrib/AutoNUE)
+  * [Domain Adaptation](./contrib/DomainAdaptation)
 
-System Requirements:
-* PaddlePaddle >= 2.0.0
-* Python >= 3.6+
 
-Highly recommend you install the GPU version of PaddlePaddle, due to the large overhead of segmentation models, otherwise, it could be out of memory while running the models. For more detailed installation tutorials, please refer to the official website of [PaddlePaddle](https://www.paddlepaddle.org.cn/install/quick?docurl=/documentation/docs/zh/2.0/install/)。
+## Community <img src="./docs/images/chat.png" width="30"/>
 
+* If you have any problem or suggestion on PaddleSeg, please send us issues through [GitHub Issues](https://github.com/PaddlePaddle/PaddleSeg/issues).
+* Welcome to Join PaddleSeg QQ Group
+<div align="center">
+<img src="./docs/images/QQ_chat.png"  width = "200" />  
+</div>
 
-#### step 2. Install PaddleSeg
-Support to construct a customized segmentation framework with the *API Calling* method for flexible development.
-
-```shell
-pip install paddleseg
-```
-
-
-#### step 3. Download PaddleSeg Repo
-Support to complete the whole process segmentation application with *Configuration Drive* method, simple and fast.
-
-```shell
-git clone https://github.com/PaddlePaddle/PaddleSeg
-```
-
-#### step 4. Verify Installation
-Run the following command. If you can train normally, you have installed it successfully.
-
-```shell
-python train.py --config configs/quick_start/bisenet_optic_disc_512x512_1k.yml
-```
-
-
-## Practical Cases
-
-* [PP-HumanSeg](./contrib/PP-HumanSeg)
-* [Cityscapes SOTA](./contrib/CityscapesSOTA)
-* [Panoptic Segmentation](./contrib/PanopticDeepLab)
-* [CVPR Champion Solution](./contrib/AutoNUE)
-* [Interactive Segmentation](./EISeg)
-* [Matting](./contrib/Matting)
-* [Domain Adaptation](./contrib/DomainAdaptation)
+## License
 
-## Feedbacks and Contact
-* The dynamic version is still under development, if you find any issue or have an idea on new features, please don't hesitate to contact us via [GitHub Issues](https://github.com/PaddlePaddle/PaddleSeg/issues).
-* PaddleSeg User Group (QQ): 1004738029 or 850378321 or 793114768
+PaddleSeg is released under the [Apache 2.0 license](LICENSE).
 
 ## Acknowledgement
 * Thanks [jm12138](https://github.com/jm12138) for contributing U<sup>2</sup>-Net.
```

Comment:
change 314 as well "ESPNet" to "ESPNetV2"

Reply:
done